src/compas_rhino/artists/artist.py

```diff
@@ -18,7 +18,7 @@
 
 
 class Artist(object):
-    """The base ``Artist`` defines functionality for drawing geometric primitives in Rhino.
+    """The base ``Artist`` defines functionality for drawing a bounded datastructure in Rhino.
```

Comment:
should we then not create a base artist called `Artist` and then do

* `Artist` > `PrimitiveArtist` > `LineArtist`, ...
* `Artist` > `DataStructureArtist` > `MeshArtist`, ...

Reply:
mmmm.....agree, to be honest I feel that's a much better idea